spec/models/user_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+Monban.test_mode!
+
+describe User do
+  describe 'validations' do
+    it { should validate_presence_of(:email) }
+    it { should validate_presence_of(:password_digest) }
+  end
+
+  describe "when email format is invalid" do
+    it "should be invalid" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.